### ISSUE TITLE
Error: Running LUA method 'update'... bad argument #1 to 'remove' (table expected, got nil)

### ISSUE THREAD

**Issue by ph1go** [Courseplay/Courseplay_FS22#75]
I created a course to harvest a field (a combination of 68 and 69 on Elmscreek), the combine goes straight past the start point without unfolding, goes in a straight line for about a hundred yards and then stops, saying it's blocked by an object.

Log here: https://pastebin.com/raw/1vFtU7W5

Screenshots:
![fsScreen_2021_12_25_12_11_11](https://user-images.githubusercontent.com/30246276/147383742-6ae900e5-2c57-4df6-8928-17361d7e47c6.jpg)
![fsScreen_2021_12_25_12_02_06](https://user-images.githubusercontent.com/30246276/147383656-4c2e356d-7b87-498b-ac38-3ee94f984ec7.jpg)
![fsScreen_2021_12_25_12_02_13](https://user-images.githubusercontent.com/30246276/147383664-ce3035ca-713d-4d33-bccf-4c6df9f65521.jpg)

**Comment by pvaiko** [comment]
Duplicate of #41 and #35.
